src/LightningExtension/Context/EntityBrowserContext.behat.inc

```diff
@@ -106,4 +107,40 @@ class EntityBrowserContext extends DrupalSubContextBase {
     $items[$n - 1]->click();
   }
 
+  /**
+   * Asserts a certain number of items are visible in the entity browser.
+   *
+   * @param int $n
+   *   The number of media items that should be visible.
+   * @param string $browser_id
+   *   (optional) The entity browser ID.
+   *
+   * @throws ExpectationException
+   *   If actual number of in the entity browser does not match the expected.
+   *
+   * @Then I should see :n items in the entity browser
```

Comment:
Let's support single-count checks by making this: "I should see :n item(s) in the entity browser"